text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/MiniMessage.java

```diff
@@ -185,7 +185,7 @@ static Builder builder() {
      * @return this builder
      * @since 4.10.0
      */
-    @NotNull Builder debug(final @Nullable Appendable debugOutput);
+    @NotNull Builder debug(final @Nullable Consumer<String> debugOutput);
```

Comment:
Documentation should mention how this is used - I suspect some people (like myself at first) will assume this is an *entire* message per-string.